api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/DateTimeIntervalValidationVTwo.java

```diff
@@ -195,7 +195,7 @@ private Map<String, String> checkDateLiterals(
                  Calendar endCal = endDate.asCalendar();
 
                  if( endCal != null ){
-                     if( !startCal.before( endCal ) ){
+                     if (!startCal.before(endCal) && !startCal.equals(endCal)) {
```

Comment:
Can this be simplified by 
```suggestion
                     if (startCal.after(endCal)) {
```